docs/man/nutdrv_qx.txt

```diff
@@ -324,7 +324,7 @@ The argument is a regular expression that must match the bus name where the UPS
 
 *subdriver =* 'string'::
 Select a serial-over-USB subdriver to use.
-You have a choice between *cypress*, *fabula*, *fuji*, *hunnox*, *ippon*, *krauler*, *phoenix* and *sgs*.
+You have a choice between *cypress*, *fabula*, *fuji*, *hunnox*, *ippon*, *krauler*, *phoenix*, *sgs* and *sgs*.
```

Comment:
Double *sgs*

Reply:
Fixed in master branch after merge, thanks for bringing the attention to this and sorry for the typo.